prime-router/docs/design/proposals/0025-handling-missing-data.md

```diff
@@ -0,0 +1,36 @@
+# Proposal for handling missing timezones in messages
+
+## Background
+
+ReportStream is currently facing an ongoing issue regarding missing timezones in date times sent by some senders. 
+Certain STLTs require the conversion of all datetime values to their respective local timezones. In cases where 
+timezones are absent, ReportStream defaults the datetime to UTC and then converts it to the receiver's local timezone.
+This can be an issue since ReportStream is essentially making up data and could be reporting incorrect data.
+To tackle this issue, there are alternative approaches to handling missing timezones instead of resorting to UTC 
+as the default. We also propose implementing a logging mechanism to notify the STLT when such discrepancies occur.
+
+## Possible Implementations
+
+We should encourage Senders to include timezones for date times whenever they're missing from messages and issue 
+a warning message whenever this information is absent, ensuring they are aware of the issue.
+
+In cases where a sender cannot provide a timezone for a datetime, ReportStream will need to determine the most appropriate timezone for the Sender. This can be achieved through the following methods:
+
+* Request the sender to specify a default timezone for their date times, which could be added as a sender transform.
```

Comment:
Can we ask STLTs do they want us to reject the message if the sender can't provide the timezone?

Reply:
Added it as a bullet point